daphne/benches/aggregation.rs

```diff
@@ -45,10 +45,10 @@ fn handle_agg_job_init_req(c: &mut Criterion) {
         });
 
         c.bench_function(&format!("handle_agg_job_init_req {vdaf:?}"), |b| {
+            // XXX Benchmarks have "improved" by 1000%. Obviously there is some sort of measurement
+            // issue going on...
```

Comment:
@mendess I'm wondering if you have any experience with criterion that might shed light here.

Reply:
Bump.

Reply:
Conclusion: https://github.com/cloudflare/daphne/issues/368